Library/Formula/joshua.rb

```diff
@@ -0,0 +1,55 @@
+class Joshua < Formula
+  desc "Statistical machine translation decoder."
+  homepage "http://joshua-decoder.org/"
+  url "https://cs.jhu.edu/~post/files/joshua-6.0.5.tgz"
+  sha256 "972116a74468389e89da018dd985f1ed1005b92401907881a14bdcc1be8bd98a"
+  head "https://github.com/joshua-decoder/joshua.git"
+
+  option "with-language-packs", "Build with all available community language packs (es-->en and ar-->en)"
```

Comment:
I think it's worth just having these as separate options and allowing them both to be specified.